apps/framework-docs/src/pages/deploying/summary.mdx

```diff
@@ -1,11 +1,12 @@
 # Deploying Moose into production
 
-Once you've finished developing your Moose application locally, the next step is to deploy your Moose app into production. Check out the Architecture page to see what the architecture of your Moose application in production will look like. 
+Once you've finished developing your Moose application locally, the next step is to deploy your Moose app into production. Check out the [Architecture page](../getting-started/architecture.mdx) to see what the architecture of your Moose application in production will look like.
```

Comment:
I added a link to the Architecture page because other pages have cross-links as well.